learning-development/learning-support.qmd

```diff
@@ -48,23 +53,76 @@ In this workshop, we cover:
 
-We also have specific learning resources and materials for [SQL](../learning-development/sql.html), [R](../learning-development/r.html), and [Git](../learning-development/git.html)
+Partnerships are designed as short-term projects, agreed in conjunction with the team and their Grade 6. The maximum length for the majority of partnership programmes will be three months, and the project must be clearly defined at the outset.
+
+The aim of partnership programmes is always to upskill the team throughout the project. For this reason, we require any teams taking part to commit time to upskilling during the partnership and being willing to independently continue the work after the partnership is complete, if appropriate. 
+
+---
+
+### What could a partnership programme involve?
+
+---
+
+- Flexible skilled resource for your team to support with R, Git, SQL or Databricks
+- Help with dedicated upskilling 
+- Knowledge sharing, including around best practice and accessibility, and peer review
+- Support with RAP implementation to allow your team to meet the RAP baseline or best practice criteria
+
+For example, this might be:
 
+- Automating processes such as data manipulation or QA
+- Creation of R Shiny dashboards
+- Reviewing existing analytical code
+- Support with moving to Azure DevOps or GitHub
 
 ---
 
-# General resources
+### What types of partnership are there?
 
 ---
 
+We have four levels of support available:
+
+- One-off bespoke support via a team meeting or a workshop, or one-off peer review 
+- **Tier 1**: occasional check-in meetings to provide best practice guidance 
+- **Tier 2**: hands on support for teams with skills or resource gaps 
+- **Tier 3**: designated resource working within your team to write code / complete a project for you, designed for complex projects
+
+Teams participating in tiers 1-3 will be assigned a named person from the Statistics Development Team who will support you in completing the project. 
+
+---
+
+### What is the process to get started with a partnership?
+
+---
+
+Once you have a clear, defined piece of work that you think would be appropriate for a partnership programme, contact [statistics.development@education.gov.uk](mailto:statistics.development@education.gov.uk). You will be sent a form to complete so that you can outline your project and requirements from us. Once we have received your completed form, we'll have an introductory meeting with your team to discuss the project and agree an appropriate tier and timescales. We will then assess your needs, determine who will be best placed to complete the work, and prioritise it according to our existing workload. Your team will be kept updated throughout the process.
+
+::: {.callout-important}
+Please note: all new tier 3 partnerships will require sign off at Grade 6 level 
```

Comment:
I think you should say why - 
to ensure prioritisation of skills development in parallel where needed? 
to ensure handover plan at the end?

Reply:
Added

Reply:
Nice - resolving